source/_posts/2020/2020-01-27-switch-travis-to-github-actions-to-reduce-stress.md

```diff
@@ -208,6 +208,27 @@ jobs:
             -   run: composer check-cs
 ```
 
+If you commit the `composer.lock` file to your repository, you can even cache composer dependencies, so it will download only when they're changed or updated.
+
+```yaml
+# .github/workflows/code_checks.yaml 
+jobs:
+    ecs:
+    # ...
+        runs-on: ubuntu-latest
+        steps:
+        - name: Load Composer dependencies from cache
+              id: composer-cache
+              run: |
+                  echo "::set-output name=dir::$(composer config cache-files-dir)"
+        - uses: actions/cache@v1
+          with:
+              path: ${{ steps.composer-cache.outputs.dir }}
+              key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+              restore-keys: |
+                ${{ runner.os }}-composer-
```

Comment:
That mighe be a topic for a new post (fancy PR to @pehapkari blog? :)). 2 minutes feedback are good enough now and this is quite huge code polution of every job that uses compose install.

Do you know of some single line way?

Reply:
Not yet. I just found a quick solution, because I didn't want to download all dependencies all the time, this is faster.

I'm noting it aside as a topic for an article :)